python/ray/tests/test_dynres.py

```diff
@@ -9,6 +9,14 @@
 logger = logging.getLogger(__name__)
 
 
+def wait_for_condition(condition_predictor):
```

Comment:
why introduce this layer of indirection? better to just call the existing function IMO

Reply:
Because the default timeout of the original function is too large.

Reply:
Sure, but why do we need to introduce a new function for this? Would be better to just call the underlying `wait_for_condition`. Also, having a long timeout shouldn't be an issue in the common (passing) case and often the CI can be extremely slow.

Reply:
why not just setting a shorter timeout? It seems to be configurable?

Reply:
I just wanted to avoid configuring timeout everywhere. I think this new function is pretty straightforward, so it shouldn't cause any confusion. 
That said, it's minor though. I can change this to use the original one.